openassessment/xblock/staff_area_mixin.py

```diff
@@ -343,6 +343,10 @@ def get_student_submission_context(self, student_username, submission):
             'student_username': student_username,
         }
 
+        if submission['file_url'] is not None:
```

Comment:
Definitely. This actually caught me when making the fix - I forgot to include `file_upload_type` initially and it took a few minutes to realize why it wasn't appearing.

I have a few ideas to clean this section up a bit, hang on.